.circleci/config.yml

```diff
@@ -209,7 +209,21 @@ commands:
           # further blocked by issue #6897.
           name: run tests
           environment:
-            GALLIUM_DRIVER: softpipe # TODO: use the default llvmpipe when it supports more extensions
+            # There are tests in the browser test suite that use libraries
+            # that are not included by "./embuilder build ALL".  For example the
+            # PIC version of libSDL which is used by test_sdl2_misc_main_module
+            EM_FOO: "0"
+            FOO: "0"
+            EM_BAR: "1"
+            BAR: "1"
+            EM_FROZEN: "0"
+            FROZEN: "0"
+            EM_XX: "0"
+            FROZEN: "0"
```

Comment:
What are all these env vars?

Reply:
Sorry just debugging .. seeing some very odd behavior on circle ci where it is seemingly ignoring some env vars.